yarn-project/circuit-types/src/l2_block.ts

```diff
@@ -186,17 +218,6 @@ export class L2Block {
     return sha256(inputValue);
   }
 
-  /**
-   * Compute the hash of all of this blocks l1 to l2 messages,
-   * The hash is also calculated within the contract when the block is submitted.
-   * @returns The hash of all of the l1 to l2 messages.
-   */
-  getL1ToL2MessagesHash(): Buffer {
-    // Create a long buffer of all of the l1 to l2 messages
-    const l1ToL2Messages = Buffer.concat(this.body.l1ToL2Messages.map(message => message.toBuffer()));
-    return sha256(l1ToL2Messages);
-  }
```

Comment:
Reverting this accidental change https://github.com/AztecProtocol/aztec-packages/pull/5371.